### ISSUE TITLE
[New Spreadsheet Layer]: test spreadsheet

### ISSUE THREAD

**Issue by jywarren** [daemon1024/leaflet-environmental-layers#12]
## New Spreadsheet Layer

Please describe the purpose of this layer

## Layer Data

```json
{"name":"Sites of interest","url":"https://docs.google.com/spreadsheets/d/1J9rqD-JfbGt5AdBYIpkeBZxetURSnIjlajkEX-aYvSk/edit?usp=sharing"}
```

Testing this out! Screenshots we could use in documentation:

![Screen Shot 2021-07-13 at 10 40 45 AM](https://user-images.githubusercontent.com/24359/125472243-13cc2f67-e969-49bd-95b6-608af5a3f257.png)

![Screen Shot 2021-07-13 at 10 40 35 AM](https://user-images.githubusercontent.com/24359/125472244-342e5e08-a26e-4b9c-9410-61643152e7c8.png)

![Screen Shot 2021-07-13 at 10 41 09 AM](https://user-images.githubusercontent.com/24359/125472240-1e784e75-13cc-4e8e-aad2-f093df18be7d.png)

**Comment by jywarren** [comment]
Hi @daemon1024 - just trying this out! Maybe we can also have a spreadsheet template people can copy, although it should be pretty easy right?

https://docs.google.com/spreadsheets/d/1J9rqD-JfbGt5AdBYIpkeBZxetURSnIjlajkEX-aYvSk/edit#gid=0

**Comment by daemon1024** [comment]
Added error handling, and the script will comment when action fails. 
https://github.com/daemon1024/leaflet-environmental-layers/issues/16

Currently uses my account since it's my access token, can move on to a bot account later.

**Comment by daemon1024** [comment]
Things work fine now.
https://github.com/daemon1024/leaflet-environmental-layers/pull/19

Wondering why we have the dist file in gitignore since it's an important part of our repo I believe and that's what we are publishing essentially.

**Comment by daemon1024** [comment]
> Maybe we can also have a spreadsheet template people can copy, although it should be pretty easy right?
> 
Definitely, We can include it as part of documentation. I will note it down in the web page as well.

**Comment by jywarren** [comment]
> Wondering why we have the dist file in gitignore since it's an important part of our repo I believe and that's what we are publishing essentially.

I believe it's because we wanted to have it compiled only when we intend to - it makes it easier to merge PRs - and so we do it manually especially before releasing new versions on NPM.
